plugins/inventory/vmware_vm_inventory.py

```diff
@@ -660,6 +682,18 @@ def _populate_host_properties(self, host_properties, host):
 
         # Use constructed if applicable
         strict = self.get_option('strict')
+
+        # Composed variables
+        compose = self.get_option('compose')
+        if not compose:
+            compose['ansible_host'] = 'guest.ipAddress'
```

Comment:
noting for posterity the option from the old script

https://github.com/ansible/ansible/blob/stable-2.9/contrib/inventory/vmware_inventory.ini#L59-L61

`host_pattern={{ guest.ipaddress }}`